packages/edit-site/src/components/add-new-template/add-custom-template-modal.js

```diff
@@ -76,48 +87,73 @@ function SuggestionListItem( {
 	);
 }
 
-function SuggestionList( { entityForSuggestions, onSelect } ) {
-	const composite = useCompositeState( { orientation: 'vertical' } );
-	const [ suggestions, setSuggestions ] = useState( EMPTY_ARRAY );
-	// We need to track two values, the search input's value(searchInputValue)
-	// and the one we want to debounce(search) and make REST API requests.
-	const [ searchInputValue, setSearchInputValue ] = useState( '' );
-	const [ search, setSearch ] = useState( '' );
-	const debouncedSearch = useDebounce( setSearch, 250 );
-	const query = {
-		...BASE_QUERY,
-		search,
-		orderby: search ? 'relevance' : 'modified',
-		exclude: entityForSuggestions.postsToExclude,
-		per_page: search ? 20 : 10,
-	};
+function useDebouncedInput() {
```

Comment:
I am a bit biased, and my goal with the refactoring with `useDebouncedInput` and `useSearchSuggestions` was to rewrite in a way that doesn't need comments to be easy to understand. So: is it succeeding? Or should these hooks be described?

Reply:
I think it can be addressed as part of: https://github.com/WordPress/gutenberg/pull/41875#issuecomment-1172561934 as well. Personally I know exactly what it does, so I might not be the best candidate to judge as well.